stable/elasticsearch-exporter/README.md

```diff
@@ -72,6 +72,7 @@ Parameter | Description | Default
 `service.annotations` | Annotations on the http service | `{}`
 `service.labels` | Additional labels for the service definition | `{}`
 `env` | Extra environment variables passed to pod | `{}`
+`envFromSecret` | Name of a Kubernetes secret (must be manually created in the same namespace) containing values to be added to the environment | `nil`
```

Comment:
maybe a bit shorter: "The name of an existing secret in the same kubernetes namespace which contains values to be added to the environment"

Reply:
no problem, updated